index.d.ts

```diff
@@ -507,6 +508,8 @@ declare namespace WAWebJS {
         location: Location,
         /** List of vCards contained in the message */
         vCards: string[],
+        /** Invite v4 info */
+        invite: object
```

Comment:
We should offer better typings for this object and what it contains

Reply:
Done (i think), this is my first PR 😅